Sources/MemoryStore.swift

```diff
@@ -61,17 +59,22 @@ class MemoryStore: Store {
         return sanitizedEntity
     }
 
-    func findAll(type: Model.Type, callback: @escaping EntitiesCallback) {
+    var entities: [[String:Any]] = []
+    var nextId: Int = 1
+
+    public init() {}
+
+    public func findAll(type: Model.Type, callback: @escaping EntitiesCallback) {
         let matchingEntities = entities.filter { $0["_type"] as! Model.Type == type }
-                                       .map(MemoryStore.sanitize)
+            .map(MemoryStore.sanitize)
```

Comment:
Is there a good reason to change the indentation here, I thought it looked cleared when the functions were aligned.